documentation/asciidoc/topics/proc_upgrading_clusters_rolling.adoc

```diff
@@ -0,0 +1,28 @@
+[id='upgrading-clusters-rolling_{context}']
+= Performing Hot Rod rolling upgrades for {brandname} clusters
+
+[role="_abstract"]
+Performing Hot Rod rolling upgrades lets you move to a new {brandname} version without service disruption.
+However, this upgrade type requires additional capacity and temporarily results in two {brandname} clusters with different versions running concurrently.
+
+.Prerequisites
+
+* Disable passivation for any caches that use persistent cache store configuration, if necessary.
```

Comment:
How does the user do this?

Reply:
I've reworded this after discussing with Gustavo a little more about the need to disable passivation. I'm not sure how great this is as a prerequisite. It would probably be pretty onerous to have to recreate a cache and disable passivation.

Reply:
This can't be done at runtime. This is a limitation of the rolling upgrade process, but just when giving up and trying to go back to the original cluster. I'd issue a WARNING to avoid doing it if you have caches with passivation